CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [5.1.1-alpha.0](https://github.com/chanzuckerberg/edu-design-system/compare/v5.1.0...v5.1.1-alpha.0) (2022-09-21)
+
```

Comment:
I can't remember if we want to commit the alpha build updates or not.

Reply:
we have not been

Reply:
I feel like it's messy and I don't want to have to commit every time I try out an alpha build, but I guess one reason to do it is then it's easier to figure out which alpha build has which change if you're looking for a specific one.

Reply:
>I don't think we have been committing the alpha build updates

Cool, I'll take it out.